packages/api/wrangler.toml

```diff
@@ -34,18 +37,22 @@ route = "api-staging.web3.storage/*"
 [env.alan]
 workers_dev = true
 account_id = "4fe12d085474d33bdcfd8e9bed4d8f95"
+vars = { CLUSTER_API_URL = "https://alan-cluster-api-web3-storage.loca.lt", FAUNA_ENDPOINT = "https://graphql.fauna.com/graphql" }
 
 [env.oli]
 workers_dev = true
 account_id = "6e5a2aed80cd37d77e8d0c797a75ebbd"
+vars = { CLUSTER_API_URL = "https://oli-cluster-api-web3-storage.loca.lt", FAUNA_ENDPOINT = "https://graphql.fauna.com/graphql" }
 
 [env.yusef]
 workers_dev = true
 account_id = "8c3da25233263bd7a26c0e2e04569ded"
+vars = { CLUSTER_API_URL = "https://yusef-cluster-api-web3-storage.loca.lt", FAUNA_ENDPOINT = "https://graphql.fauna.com/graphql" }
 
 [env.vsantos]
 workers_dev = true
 account_id = "7ec0b7cf2ec201b2580374e53ba5f37b"
+vars = { CLUSTER_API_URL = "https://vsantos-cluster-api-web3-storage.loca.lt", FAUNA_ENDPOINT = "https://graphql.fauna.com/graphql" }
```

Comment:
I needed to add the FAUNDA_ENDPOINT otherwise it was undefined in the worker. @alanshaw given that this is always the same, should we make a constant out of it ?

Reply:
👍